commit 6b6885f5e13d53063ccff86d99ec934197dc0d06
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-07-03T12:56:49-06:00 (3 months ago)
    
    MANTA-4384 Electric-boray does not pass response to deleteObject rpc

diff --git a/lib/server.js b/lib/server.js
index 1b08c4c..a39ce4d 100644
--- a/lib/server.js
+++ b/lib/server.js
@@ -1098,6 +1098,7 @@ function deleteObject(options) {
                 if (gErr) {
                     rpc.fail(gErr);
                 } else {
+                    rpc.write(obj);
                     rpc.end();
                 }
             });
